ProiectPractica/Repository/RepositoryConfigurare.cs

```diff
@@ -0,0 +1,81 @@
+﻿using ProiectPractica.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Schema;
+using Newtonsoft.Json.Linq;
+
+namespace ProiectPractica.Repository
+{
+    internal class RepositoryConfigurare : IRepository
+    {
+        /// <summary>
+        ///  This function reads from a file and returns the string read from the file, if the file exists.
+        ///  And if the file does not exists the function throws an exception.
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <returns></returns>
+        /// <exception cref="Exception"></exception>
+        private string ReadFromFile(string filePath)
+        {
+            if (!File.Exists(filePath))
+            {
+                throw new Exception("File not found!");
```

Comment:
Could use a more specific exception here. Exception is a bit too general. You could create your own, but I think this one fits your needs pretty well: https://learn.microsoft.com/en-us/dotnet/api/system.io.filenotfoundexception?view=net-8.0